lang/en.php

```diff
@@ -81,12 +81,11 @@
       'gallery' => 'home'
     ],
     'join' => [
-      'title' => 'JOIN <span>US</span>',
```

Comment:
Is a (span) needed in this place? I deleted it in "pl".

Reply:
span was removed
p.s. 
red is removed, green is added